crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -24,6 +24,9 @@ def status
       end
 
       def checks
+        upgrade_status = ::Crowbar::UpgradeStatus.new
+        upgrade_status.start_step
```

Comment:
Are you assuming that the step you are starting is `upgrade_prechecks`?  If so, can you really be sure of that, given that `::Crowbar::UpgradeStatus#initialize` checks for a progress file, and if it's already there, loads the state in from it?